site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -470,7 +470,7 @@ Whether to force the user to use at least one number.
 [field]#tenant.passwordValidationRules.validateOnLogin# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.15.0#::
 When enabled the user's password will be validated during login. If the password does not meet the currently configured validation rules the user will be required to change their password.
 
-[field]#tenant.themeId# [type]#[UUID]# [required]#Required# [since]#Available since 1.8.0#::
+[field]#tenant.themeId# [type]#[UUID]# [optional]#Optional# [default]#defaults to the default tenant theme Id# [since]#Available since 1.8.0#::
```

Comment:
The default Theme Id is `75a068fd-e94b-451a-9aeb-3ddb9a3b5987` - we could optionally state this as well?